src/loaders/orgUnitLoader.js

```diff
@@ -37,11 +42,8 @@ const orgUnitLoader = async (config) => {
             .catch((error) => {
                 if (error && error.message) {
```

Comment:
API related errors do not seem to have a message property.

Reply:
Fixed